com.microsoft.mrtk.input/Interactors/Ray/MRTKRayInteractor.cs

```diff
@@ -13,6 +13,14 @@ namespace Microsoft.MixedReality.Toolkit.Input
     /// A wrapper for the XRRayInteractor which stores extra information for MRTK management/services
     /// </summary>
     [AddComponentMenu("MRTK/Input/MRTK Ray Interactor")]
+
+    // This execution order ensures that the MRTKRayInteractor runs its update function right after the
+    // XRController. We do this because the MRTKRayInteractor needs to set its own pose after the parent controller transform,
+    // but before any physics raycast calls are made to determine selection. The earliest a physics call can be made is within
+    // the UIInputModule, which has an update order much higher than XRControllers.
+    // TODO: Examine the update order of other interactors in the future with respect to when their physics calls happen,
+    // or create a system to keep ensure interactor poses aren't ever implicitly set via parenting.
+    [DefaultExecutionOrder(XRInteractionUpdateOrder.k_Controllers + 1)]
```

Comment:
Do we want to do this for all our interactors that update their own pose, to be safe?

Reply:
Oh, just saw the TODO

> // TODO: Examine the update order of other interactors in the future with respect to when their physics calls happen,
    // or create a system to keep ensure interactor poses aren't ever implicitly set via parenting.

Reply:
Yeah I don't want to do them all now because the current visible effects are minor and the potential new issues introduced could be large...